.github/workflows/Coverage.yaml

```diff
@@ -0,0 +1,48 @@
+name: Coverage
+
+on: [pull_request]
+
+jobs:
+  coverage:
+    runs-on: macos-latest
+    timeout-minutes: 10
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+          ref: ${{ github.event.pull_request.head.ref }}
+
+      - name: Setup Python
+        uses: actions/setup-python@v4 
+        with:
+          python-version: '3.9'
+          cache: 'pip'
+
+      - name: Install Python dependencies
+        run: pip install -r requirements.txt
+
+      - name: Install Slather
+        run: gem install slather
```

Comment:
Previously, I tried using a `Gemfile` and the [setup-ruby](https://github.com/ruby/setup-ruby) action. Unfortunately, this didn't modify the `PATH` as promised so I fell back to this comparatively simple approach.